apidef/oas/oas.go

```diff
@@ -21,6 +21,7 @@ func (s *OAS) Fill(api apidef.APIDefinition) {
 	}
 
 	xTykAPIGateway.Fill(api)
+	s.fillPathsAndOperations(api.VersionData.Versions[""].ExtendedPaths)
```

Comment:
Reading this bit of code, it isn't clear to me why "" was chosen and what value it represents. Can we do a constant for the keyname, so it becomes Versions[???]

Reply:
correct.